### ISSUE TITLE
camel-python extension

### ISSUE THREAD

**Issue by davsclaus** [apache/camel-quarkus#4408]
Camel 3.19 comes with python and it would be good to have an extension for this

**Comment by davsclaus** [comment]
@essobedo you are doing great with the language extensions, here is another we need

**Comment by Shahid-Roofi-Khan** [comment]
+1 also javascript

**Comment by essobedo** [comment]
@davsclaus Good idea, but we need a Python DSL first to be able to have it in Camel-K

**Comment by davsclaus** [comment]
Not at first its a scripting language not for DSL

**Comment by kulagaIA** [comment]
Hello! Does anyone work on this? I would like to make camel-quarkus-python extension.

**Comment by kulagaIA** [comment]
@essobedo please tell if you have any work done on this issue. If not, I will start working on it.

**Comment by essobedo** [comment]
@kulagaIA no I don't, you can do the task, thx for asking
